tests/inputs/TestNetworkManager.cpp

```diff
@@ -104,3 +104,17 @@ TEST(NetworkManager, shunts) {
   manager.walkNodes();
   ASSERT_EQ(nbShunts, 1);
 }
+
+TEST(NetworkManager, initialConditions) {
+  using dfl::inputs::NetworkManager;
+
+  NetworkManager manager("res/initial_conditions_OK.iidm");
+
+  ASSERT_TRUE(manager.isFullyConditioned());
+  ASSERT_TRUE(manager.isPartiallyConditioned());
+
+  NetworkManager manager2("res/initial_conditions_NOK.iidm");
+
+  ASSERT_FALSE(manager2.isFullyConditioned());
```

Comment:
We need a case in the middle with Partially=true and fully = false

Reply:
good point